board/ipaccess/ipa9131/sec.c

```diff
@@ -0,0 +1,43 @@
+#include <common.h>
+#include <compiler.h>
+#include <asm/errno.h>
+#include <asm/io.h>
+#include <asm/immap_85xx.h>
+#include "characterisation.h"
+
+
+#define sec_out_be32(x,y) out_be32((volatile unsigned __iomem *)(x),(y))
+
+#define SECMON_HPCOMR        (CONFIG_SYS_IMMR + 0x000e6004)
+
+
+
+static void set_sec_state(void)
```

Comment:
ohh it does work writing all the bits together just confirmed it.. I had a notion(while I was working on secjr driver) that writing all together is not working just for a silly reason that I forgot I am working on a non-secure boot and writing 3 on this register at the last bit will actually transition sec engine from initial state to Trusted state..while I was wrongly expecting it to be in secure state. I will test this once on a Secure boot and change this and probably the driver code as well. 
Do we also need to check the sec engine state after this to make sure we are on secure(fro prod)/non-secure(specials) state?